tester/src/test/java/com/silanis/esl/sdk/examples/DataManagementPolicyExampleTest.java

```diff
@@ -0,0 +1,28 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.sdk.DataManagementPolicy;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class DataManagementPolicyExampleTest {
+
+    @Test
+    public void verifyResult() {
+        DataManagementPolicyExample example = new DataManagementPolicyExample();
+        example.run();
+
+        DataManagementPolicy dataManagementPolicy = example.getDataManagementPolicyAfterUpdate();
```

Comment:
This example for the integration user. It's kind of a document. If possible, avoid encapsulation and set the variable to be public so that it can be accessed directly in the example test if necessary.